File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    let arr = [];
    for (let i = 0; i < arrayToSearchThrough.length; i++) {
      if (arrayToSearchThrough[i] === valueToFind) {
        arr.push(i);
      }
    }
    return arr;
  }

  exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    let arr = [];
    for (let i = 0; i < arrayToSearchThrough.length; i++) {
      if (arrayToSearchThrough[i] === valueToFind) {
        arr.push(i);
      }
    }
    return arr;
  }

File: python/linear_search.py
#array_to_search_through = []
#for number in range(1, 1001):
#    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    # your code here
    index_match = 0
    for i, x in enumerate(array_to_search_through):
        if x == value_to_find:
            index_match = i
            return index_match
            